packages/i18n/data/es.json

```diff
@@ -15,7 +15,7 @@
   "UIKit.DataTableManager.ColumnSelectorGroups.visible": "Columnas visibles",
   "UIKit.DataTableManager.DensityManager.closeButtonLabel": "Cerrar",
   "UIKit.DataTableManager.DensityManager.densityDisplayCompactOption": "Compacta",
-  "UIKit.DataTableManager.DensityManager.densityDisplayDefaultOption": "Predeterminada",
+  "UIKit.DataTableManager.DensityManager.densityDisplayComfortableOption": "Confortable",
```

Comment:
What's the spanish translation for this? @CarlosCortizasCT

Reply:
That's the one: "Confortable"

Reply:
Cool, then we are all set? Could I merge this PR?

Reply:
I think we can, yes.

Many thanks!